code/game/machinery/cloning.dm

```diff
@@ -253,7 +253,7 @@
 		else if((mob_occupant.cloneloss <= (100 - heal_level)))
 			connected_message("Cloning Process Complete.")
 			SPEAK("The cloning cycle of [mob_occupant.real_name] is complete.")
-
+			mob_occupant.setCloneLoss(0)

```

Comment:
>doesnt even remove cloneloss, just sets it to 0 after cloning is done

🥇

Reply:
yes? it's still used internally for processing, read the fucking description you dickwad

Reply:
Doesn't even do this much properly since you should be removing the bits where the clone loss is added in the first place
Hahahaha this is so bad

Reply:
oh my god

you literally CANNOT FUCKING READ

JESUS CHRIST

Reply:
Proper coding would make it so the processing no longer relies on cloneloss, dummy